consensus/polybft/consensus_runtime.go

```diff
@@ -115,22 +117,38 @@ type consensusRuntime struct {
 	// activeValidatorFlag indicates whether the given node is amongst currently active validator set
 	activeValidatorFlag uint32
 
+	// checkpointsOffset represents offset between checkpoint blocks (applicable only for non-epoch ending blocks)
+	checkpointsOffset uint64
```

Comment:
I would move this `checkpointOffse` to `checkpointManager`, so that all checkpoint related configuration resides there.

Reply:
Fixed [6a2a52b](https://github.com/0xPolygon/polygon-edge/pull/861/commits/6a2a52ba4577cd8c48852e9273e77ae3fc5492fe)